docker-compose.yml

```diff
@@ -125,22 +199,29 @@ services:
       - edxapp_studio_assets:/edx/var/edxapp/staticfiles/
 
   forum:
-    image: hello-world
-    command: /hello
+    command: bash -c 'source /edx/app/forum/ruby_env && source /edx/app/forum/devstack_forum_env && cd /edx/app/forum/cs_comments_service && bundle install && while true; do ruby app.rb -o 0.0.0.0 ; sleep 2; done'
+    container_name: edx.devstack.forum
+    depends_on:
+      - mongo
+      - memcached
+      - elasticsearch
+    image: edxops/forum:hawthorn.master
+    ports:
+      - "44567:4567"
 
   postgres:
     image: postgres:9.4
     container_name: tahoe.devstack.postgres
     volumes:
       - postgres_data:/var/lib/postgresql/data
     ports:
-      - ${TAHOE_POSTGRES_PORT-5432}:5432
+      - ${TAHOE_POSTGRES_PORT}:5432
```

Comment:
```suggestion
      - ${TAHOE_POSTGRES_PORT-5432}:5432
```

Reply:
I'm not familiar with docker compose variables. Why do we want to specify the port in the variable name or am I reading this wrong?